package.json

```diff
@@ -7,9 +7,11 @@
     "dotenv": "^8.0.0",
     "graphql": "^14.4.2",
     "graphql-tag": "^2.10.1",
+    "i": "^0.3.6",
```

Comment:
I looked up this `i` package on NPM. It's for `inflect`, a package that manipulates strings to camel case, uppercase etc... I was wondering why is this included in our project?